decidim-core/lib/decidim/exporters.rb

```diff
@@ -6,8 +6,13 @@ module Exporters
     autoload :JSON, "decidim/exporters/json"
     autoload :CSV, "decidim/exporters/csv"
     autoload :Excel, "decidim/exporters/excel"
+    autoload :PDF, "decidim/exporters/pdf"
     autoload :ExportData, "decidim/exporters/export_data"
 
+    # i18n-tasks-use t('decidim.admin.exports.formats.JSON')
```

Comment:
can we remove this comments?

Reply:
These comments are necessary for the i18n normalizer to locate strings not used in views

Reply:
```suggestion
    # Necessary for the i18n normalizer to locate strings not directly invoked in views:
    # i18n-tasks-use t('decidim.admin.exports.formats.JSON')
```